example/bottle/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bugsnag Python: Bottle example</title>
+</head>
+
+<body>
+  <div>
+    <h1>Bugsnag Python: Bottle example</h1>
+  </div>
+  <div>
+    <h3>This page is a basic example of how to include bugsnag-python in a bottle app.
+    </h3>
+  </div>
+
+    <br>
+  <h3>Click below to send errors to your Bugsnag dashboard.</h3>
+  <br>
+  <br>
+    <div id="python">
+        <a href="/crash" style="text-decoration:none;">
+          <button type="button">UNHANDLED ERROR</button>
+        </a>
+        <br>
+        <a href="/handled" style="text-decoration:none;">
+          <button type="button">HANDLED ERROR</button>
+        </a>
+    </div>
+<!-- review settings.py & views.py to see how bugsnag integrates with your app. -->
```

Comment:
Should this be a comment? Perhaps it would be a good idea to make this an actual paragraph so readers of the rendered HTML page can view the insight.